csrankings-s.csv

```diff
@@ -355,7 +355,7 @@ Sara Mostafavi,University of Washington,http://saramostafavi.github.io/,nBL0J6kA
 Sara Nadi,University of Alberta,https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/sarah-nadi,PxNO2g8AAAAJ
 Sara Rampazzi,University of Florida,https://sararampazzi.com,I9d0CrAAAAAJ
 Sara Shurin,Hebrew University of Jerusalem,http://www.cs.huji.ac.il/~sara,J2sg9BwAAAAJ
-Sara Silva,Universidade de Lisboa,https://ciencias.ulisboa.pt/perfil/sgsilva,g15oqi0AAAAJ
+Sara Silva,Universidade de Lisboa,https://ciencias.ulisboa.pt/perfil/sgsilva,g15oqi0AAAAJ

```

Comment:
Web page is dead; not sure what commit removed this but please remove.

Reply:
I've removed the entry. The website worked for me though, so I'm not sure if the entry shouldn't stay.

Reply:
Worked for me this time, 🤷
Please restore, sorry for the churn!

Reply:
I've submitted this commit as a new PR, which seems to me to be the way to do it with the least confusing git history.